cookbook/core/control_flow/run_conditions.py

```diff
@@ -173,6 +173,12 @@ def bool_input_wf(b: bool) -> int:
     return conditional("test").if_(b.is_true()).then(success()).else_().then(failed())
 
 
+# %%
+# Also it is possible to arbitrarily nest conditional sections, inside other
+# conditional sections. Remember - conditional sections can only be in the
+# `then` part for a previous conditional block
+# The follow example shows how you can use float comparisons to create
+# a multi-level nested workflow
 @workflow
 def so_nested(my_input: float) -> float:
```

Comment:
better wf name please?  I just wrote down the first thing that came to mind when i was doing this, bad name.